webapp/helper.py

```diff
@@ -20,28 +20,33 @@ def get_or_create_user_id(user):
     return user_exists.id
 
 
-def create_jira_task(app, task):
+def create_jira_task(app, body):
     """
     Create a new issue on jira and add a record to the db
     """
     # TODO: If an epic already exists for this request, add subtasks to it.
 
     jira = app.config["JIRA"]
+    summary = ""
+    if "summary" in body.keys():
+        summary = body["summary"]
```

Comment:
We could do this in one step since in python dict attributes can optionally return `None` e.g.
```python
. . .
        description=body["description"],
        summary=body.get("summary") or "",
    )
```

Reply:
thanks for pointing this out. This remark actually led me to find another bug related to a summary in case user has not entered anything in the Summary field on UI. I'm fixing it now and will push a change soon.

Reply:
fixed this part together with the bug for supporting default titles and storing them in DB jira_tasks